### ISSUE TITLE
Ajouter une devinette pour accéder à la vidéo d'accueil.

### ISSUE THREAD

**Issue by larnoult** [parlemonde/1village#419]
En début d'année, nous envoyons une lettre aux classes participantes, pour leur faire deviner un mot "secret". 
Nous les invitons ensuite à se rendre sur 1v.parlemonde.org (la page d'accueil) afin de renseigner le mot qu'ils ont trouvé et découvrir la vidéo de lancement du projet. 

Il faudrait donc modifier la page d'accueil afin de "masquer" la vidéo avec un formulaire tel que [présenté sur cette maquette](https://www.figma.com/proto/YJX7aaNsrc4DNjRlCoqOVx/1V-2021-22?node-id=0%3A1&viewport=13024%2C-13700%2C0.3&scaling=min-zoom&starting-point-node-id=5068%3A44784&show-proto-sidebar=1). 

Les mots acceptés pour "révéler" la vidéo sont : 
{Curiosité, curiosité, curiosite, Curiosite, CURIOSITE} et le même ensemble de mot avec devant {La, la ou LA} et derrière {. ou ! ou ?}
Autrement dit, le mot _curiosité_ ou le groupe de mot _la curiosité_,  qui n'est ni case ni cute sensible, et qui peut être suivi d'une ponctuation.

**Comment by larnoult** [comment]
Super ! Est-ce possible de fixer le padding (entre le logo et la question) afin d'afficher d'avoir à scroller pour appuyer sur "envoyer" sur des grands écrans ? Je pense que ça sera illisible sur Tableau Blanc interactif par exemple.
<img width="1221" alt="image" src="https://user-images.githubusercontent.com/20889221/188154130-fc40e2e7-0c97-4f8f-9889-ea79f73cde49.png">

**Comment by larnoult** [comment]
Peux-tu également changer le "Indice" de la boite en "Votre réponse" ? Merci !

**Comment by Houra-Am** [comment]
> Super ! Est-ce possible de fixer le padding (entre le logo et la question) afin d'afficher d'avoir à scroller pour appuyer sur "envoyer" sur des grands écrans ? Je pense que ça sera illisible sur Tableau Blanc interactif par exemple. Voilà la capture sur un écran 24" <img alt="image" width="1221" src="https://user-images.githubusercontent.com/20889221/188154130-fc40e2e7-0c97-4f8f-9889-ea79f73cde49.png">

Yes

**Comment by larnoult** [comment]
Peut-on changer le texte d'erreur : 
"Ce n’est pas le bon indice" -> "Il s'agit d'une autre qualité, essaye à nouveau". 
<img width="1358" alt="image" src="https://user-images.githubusercontent.com/20889221/190340844-7239dcff-35cc-47fc-9819-456f4739212b.png">

**Comment by larnoult** [comment]
@Houra-Am le texte d'erreur affiché n'a pas encore été changé.

**Comment by larnoult** [comment]
Par ailleurs, le problème de padding est réglé sur staging.. mais est présent sur le live (bizarre ?).

**Comment by Houra-Am** [comment]
Oui, j'ai remarqué ça ! C'est bizarre effectivement. Je vais le corriger.

**Comment by yudino** [comment]
@larnoult Le texte est corrigé : 
![image](https://user-images.githubusercontent.com/75479164/192964432-3b325134-ab99-4561-8ebb-53588f53d0ba.png)

**Comment by larnoult** [comment]
@yudino Le texte d'erreur est à nouveau mauvais sur la prod. 
![image](https://user-images.githubusercontent.com/20889221/193762517-92e178d1-97d8-4b5e-8780-67d1b1f2bf74.png)

**Comment by yudino** [comment]
Elle fonctionne en prod :
![image](https://user-images.githubusercontent.com/75479164/194091957-016d71cd-0157-4e40-9d1c-f8c471462a96.png)
